java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -385,18 +383,43 @@ class DatabaseImpl(
         return typeMap
     }
 
+    private fun Long.toBoolean() = when (this) {
+        0L -> false
+        1L -> true
+        else -> throw IllegalArgumentException(
+            "Could not convert $this to Boolean, expected 0 or 1."
+        )
+    }
+
+    private fun Boolean.toLong() = when (this) {
+        false -> 0L
+        true -> 1L
+    }
+
+    private fun Cursor.getBoolean(i: Int) = getLong(i).toBoolean()
+
+    private fun SQLiteProgram.bindBoolean(i: Int, value: Boolean) = bindLong(i, value.toLong())
+
+    /** The type of the data stored at a storage key. */
+    @VisibleForTesting
+    enum class DataType {
+        ENTITY,
+        SINGLETON,
+        COLLECTION
```

Comment:
IIRC, kotlin style suggests using `CapitalCamelCase` for enum values.

Reply:
Done